src/drawers/areaDrawer.ts

```diff
@@ -21,3 +22,34 @@ export class AreaSVGDrawer extends SVGDrawer {
     return super.getVisualPrimitiveAtIndex(0);
   }
 }
+
+export function makeAreaCanvasDrawStep(
+    d3AreaFactory: () => d3.Area<any>,
+    d3LineFactory: () => d3.Line<any>,
+): CanvasDrawStep {
+  return (context: CanvasRenderingContext2D, data: any[][], attrToAppliedProjector: AttributeToAppliedProjector) => {
+    const d3Area = d3AreaFactory();
+    context.save();
+    context.beginPath();
+    d3Area.context(context);
+    d3Area(data[0]);
+    context.lineJoin = "round";
+    const areaStyle = resolveAttributes(attrToAppliedProjector, ["fill", "opacity"], data[0], 0);
+    areaStyle['opacity'] = areaStyle['opacity'] ? areaStyle['opacity'] : 1;
+    areaStyle['opacity'] *= 0.25;
+    styleContext(context, areaStyle);
+    context.restore();
+
+    const d3Line = d3LineFactory();
```

Comment:
We should call lineCanvasDrawStep here instead of copy/paste. The goal should be composition of draw steps